Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-rails (3.8.2)
+    rspec-rails (3.8.3)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       railties (>= 3.0)
       rspec-core (~> 3.8.0)
       rspec-expectations (~> 3.8.0)
       rspec-mocks (~> 3.8.0)
       rspec-support (~> 3.8.0)
-    rspec-support (3.8.2)
-    rubocop (0.74.0)
+    rspec-support (3.8.3)
+    rubocop (0.78.0)
```

Comment:
### [rubocop](https://www.rubocop.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.74.0 → 0.78.0

[_[change-log](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop-hq/rubocop/)_]





<details>
<summary>Commits</summary>

A change of **336** commits. See the full changes on [the compare page](https://github.com/rubocop-hq/rubocop/compare/v0.74.0...v0.78.0).

These are the first 10 commits:
- (f214183) [Merge pull request #7354 from koic/fix_a_false_positive_for_…](https://github.com/rubocop-hq/rubocop/commit/f214183a3390c98befc7fce86d503fc1332d6c6c)
- (c60b6b5) [[Fix #7353] Fix a false positive for `Style/RedundantSelf`](https://github.com/rubocop-hq/rubocop/commit/c60b6b5f2f98c0b7129728d054d0524b7b726e5b)
- (b63eafa) [Fix new_cop task to avoid generating invalid YAML for the co…](https://github.com/rubocop-hq/rubocop/commit/b63eafa41938bdae758eee35d9967dfd1f408648)
- (0e3258d) [Merge pull request #7357 from pocke/fix-configuration-inject…](https://github.com/rubocop-hq/rubocop/commit/0e3258d7ab13a66f07f7f75224f2880b27710455)
- (14c8aa9) [Merge pull request #7356 from koic/fix_false_positive_for_st…](https://github.com/rubocop-hq/rubocop/commit/14c8aa98942c54a90b424c4e7bb74deb7e169fa6)
- (d5ed7fe) [[Fix #7358] Fix an incorrect autocorrect for `Style/NestedMo…](https://github.com/rubocop-hq/rubocop/commit/d5ed7fe85ef65c7bb7c976ba6aa1785bb22ee1ac)
- (359f957) [[Fix #7361] Fix a false positive for `Style/TernaryParenthes…](https://github.com/rubocop-hq/rubocop/commit/359f957088a66885ec89e5e0326e89fe0adf7422)
- (e5fcc08) [Remove alias to corrected methods (#7359)](https://github.com/rubocop-hq/rubocop/commit/e5fcc08aa3bdf5e14b550f40c29e572a786e71e4)
- (3754a74) [[Fix #7369] Fix an infinite loop error for `Layout/IndentAss…](https://github.com/rubocop-hq/rubocop/commit/3754a74d504b5f04d1f4cb28b874fb31b37f9eba)
- (61a16dd) [[Fix #7345] Mark unsafe for `Style/YodaCondition`](https://github.com/rubocop-hq/rubocop/commit/61a16dda5b6314b59998b36db6ac01209b37a50e)


</details>